capi-tilt/cluster.yaml

```diff
@@ -0,0 +1,30 @@
+kind: DockerCluster
+apiVersion: infrastructure.cluster.x-k8s.io/v1alpha3
+metadata:
+  name: my-cluster
+  namespace: default
+spec:
+  controlPlaneEndpoint:
+    host: 192.168.111.249
+    port: 6443
+status:
+    ready: true
+
+---
+  
+apiVersion: cluster.x-k8s.io/v1alpha3
+kind: Cluster
+metadata:
+  name: my-cluster
+  namespace: default
+spec:
+  clusterNetwork:
+    pods:
+      cidrBlocks: ["192.168.0.0/16"]
+    serviceDomain: "cluster.local"
+  infrastructureRef:
+    apiVersion: infrastructure.cluster.x-k8s.io/v1alpha3
+    kind: DockerCluster
+    name: my-cluster
+    namespace: default
```

Comment:
Can you make it use CACPK ? You can see an example in metal3-dev-env

Reply:
Could you be more specific? Don't know exactly what to look for?

Reply:
Unfortunately it seems that CAPD does not support control plane management. [#252](https://github.com/kubernetes-sigs/cluster-api-provider-docker/issues/252)

Reply:
got this working with CACPK